project-base/storefront/components/Blocks/BlogPreview/BlogPreview.tsx

```diff
@@ -18,45 +20,47 @@ export type BlogPreviewProps = {
 export const BlogPreview: FC<BlogPreviewProps> = ({ blogArticles, blogUrl, fetchingArticles }) => {
     const { t } = useTranslation();
 
-    const blogMainItems = useMemo(
-        () => mapConnectionEdges<TypeListedBlogArticleFragment>(blogArticles?.slice(0, 2)),
-        [blogArticles],
-    );
-    const blogSideItems = useMemo(
-        () => mapConnectionEdges<TypeListedBlogArticleFragment>(blogArticles?.slice(2)),
-        [blogArticles],
-    );
+    const blogItems = useMemo(() => mapConnectionEdges<TypeListedBlogArticleFragment>(blogArticles), [blogArticles]);
+    const blogMainItems = blogItems?.slice(0, 2);
+    const blogSideItems = blogItems?.slice(2);
+
+    const { width } = useGetWindowSize();
+    const isDesktop = width > desktopFirstSizes.notLargeDesktop;
```

Comment:
I think you can achieve this easier with example from `Pagination.tsx` file - 
```typescript
const isDesktop = useMediaMin('vl');
```

Reply:
Nice, thanks for that!